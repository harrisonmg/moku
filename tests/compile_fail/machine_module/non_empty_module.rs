use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    #[machine_module]
    mod machine {
        const SOMETHING: &str = "something";
    }
}

fn main() {}
