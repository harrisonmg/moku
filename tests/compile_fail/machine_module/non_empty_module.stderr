error: a `moku::machine_module` must have empty braces, try `mod state_machine_module {}`
 --> tests/compile_fail/machine_module/non_empty_module.rs:7:5
  |
7 |     #[machine_module]
  |     ^
